test/functional/rpc_blockchain.py

```diff
@@ -108,5 +109,17 @@ def _test_getdifficulty(self):
         # binary => decimal => binary math is why we do this check
         assert abs(difficulty * 2**31 - 1) < 0.0001
 
+    def _test_getblock(self):
+        node = self.nodes[0]
+
+        # Test getblock verbosity
+        besthash = node.getbestblockhash()
+        assert(isinstance(node.getblock(blockhash=besthash, verbose=False), str))
+        assert(isinstance(node.getblock(blockhash=besthash, verbosity=0), str))
+        assert(isinstance(node.getblock(besthash, 1)['tx'][0], str))
+        assert(isinstance(node.getblock(besthash, True)['tx'][0], str))
+        assert('vin' in node.getblock(besthash, 2)['tx'][0])
```

Comment:
Nice!
I think this covers the basics

Would love to see some stricter tests then just checking if a string is returned
Maybe something like:
```
assert_is_hex_string(node.getblock(besthash, False))
assert_is_hex_string(node.getblock(besthash, 0))

assert_is_hash_string(node.getblock(besthash, True)['tx'][0]);
assert_is_hash_string(node.getblock(besthash, 1)['tx'][0]);
assert_is_hash_string(node.getblock(besthash, 2)['tx'][0]['txid']);
```